### DISCUSSION TITLE
Clearing logs (Web-UI -> "Administrator settings" -> "Logging")

### DISCUSSION THREAD

**Discussion by aga1314** [nextcloud/all-in-one#4654] [Questions] [answered]
Hello everyone,

is there an option to clear nextcloud-logs (that are shown under "Logging" in the Web-UI's "Administrator settings")?

I've forgotten to check a collabora-setting, which caused me over 400 error-entrys over the past couple days..

I'd like to clear them up / deleting as much of them as possible, to have a rather clear overview again...

Or is there a simple+safe way to just clear all logs at once?

Thanks in advance.

**Comment by szaimen** [comment, marked as answer]
Hi, you might simply clear the log by truncating the log file that is stored in `/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/data/nextcloud.log`

**Reply by adrianvanderwal** [reply]
It might be worth mentioning that additional log files (e.g. `.../nextcloud.log.1`) are created when the original file exceeds a size limit.

This might also be something worthwhile adding to an "easy first fix" list, if it exists for people looking to contribute.

**Reply by Dremor** [reply]
And where would that file size limit be ? My log file is so big my server tab freeze every time I open it. 😅

**Reply by theking2** [reply]
> Hi, you might simply clear the log by truncating the log file that is stored in `/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/data/nextcloud.log`

I don´t have such a file or folder

**Reply by IMNdi** [reply]
Mine's in \\wsl.localhost\docker-desktop\mnt\docker-desktop-disk\data\docker\volumes\nextcloud_aio_nextcloud\_data\data in case anyone's on Windows. It's most easily cleared by a small batch file.

**Comment by theking2** [comment]
how to have the nextcloud logs auto rotate?

**Reply by lumaqnhkaim** [reply]
Seems like there is an option for that: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#log-rotate-size

**Comment by bernd-wechner** [comment]
Methinks there should be an option on the web interface to either clear the log or rotate it. To essentially flag that it's been examined and I'd like the count of errors on the Administrations Settings page to drop to 0 please. That count is useless as it stands as it's just full of errors I've looked and care nought about. Like Mail apps verbose IMAP connection errors ...
